index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mini Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/toastify-js/src/toastify.min.css">
+</head>
+<body>
+    <div id="main">
+        <nav class="navbar">
+            <h1 class="navHead">PostShala</h1>
+        </nav>
+        <div class="Products"></div>
+        <div class="pagination">
+            <button class="btnPrev" disabled onclick="handleBack()">Prev</button>
+            <div class="pageBtn"></div>
+            <button class="btnNext" onclick="handleNext()">Next</button>
+        </div>
+    </div>
+    <script src="https://cdn.jsdelivr.net/npm/toastify-js"></script>
```

Comment:
read the reason, why we always prefer to put script at the bottom.

Reply:
The script is prefer to use at the bottom because :- This avoids blocking the rendering process and ensures that DOM elements are fully loaded before the script runs.